admin/sql/updates/20240221-mbs-13492.sql

```diff
@@ -0,0 +1,22 @@
+\set ON_ERROR_STOP 1
+
+BEGIN;
+
+UPDATE editor
+   SET privs = privs | 8192 -- set new beginner flag
+ WHERE id != 4 -- avoid setting ModBot as beginner
+   AND NOT deleted
+   AND ( 
+        member_since < NOW() - INTERVAL '2 weeks'
```

Comment:
Oh, I think I see the issue right after leaving the previous comment.
```suggestion
        member_since > NOW() - INTERVAL '2 weeks'
```

Reply:
Changing that gives 61,242 non-beginner editors.